src/TSTransformer/macros/identifierMacros.ts

```diff
@@ -2,6 +2,6 @@ import luau from "@roblox-ts/luau-ast";
 import { IdentifierMacro, MacroList } from "TSTransformer/macros/types";
 
 export const IDENTIFIER_MACROS: MacroList<IdentifierMacro> = {
-	PKG_VERSION: state => luau.string(state.data.pkgVersion ?? "0.0.0"),
+	$PKG_VERSION: state => luau.string(state.data.pkgVersion ?? "0.0.0"),
```

Comment:
Looking at this, I don't think this macro fits in roblox-ts. It's more something for `rbxts-transform-debug` or the like. See the same reasoning at `opcall`.